eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -350,7 +350,7 @@ stages:
 
         - template: ../jobs/vmr-build.yml
           parameters:
-            buildName: OSX
+            buildName: OSX_x64_Cross
```

Comment:
This isn't a cross-build, or is it?

Reply:
I'll just change it, it's not worth debating the philosophy over what is & isn't cross on non-Linux